.github/workflows/discovery.yaml

```diff
@@ -0,0 +1,24 @@
+on:
+  schedule:
+    - cron: '0 12 * * TUE'
+  pull_request: # REMOVE BEFORE MERGING
+  workflow_dispatch:    
+name: Update Discovery Generated Types
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
+      - uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # v4.0.2
+        with:
+          node-version: 16
```

Comment:
will these be managed by renovate or dependabot?

Reply:
yeah, renovate/dependabot managed those dependencies. See example here https://github.com/googleapis/google-api-nodejs-client/pull/3485